test/spec/unit/secureCreatives_spec.js

```diff
@@ -30,14 +30,14 @@ describe('secureCreatives', () => {
         cpm: '1.00',
         adUnitCode: 'some_dom_id'
       };
-      const remoteDomain = '*';
-      const source = {
-        postMessage: sinon.stub()
+      const event = {
+        source: { postMessage: sinon.stub() },
+        origin: 'origin.sf.com'
       };
 
-      _sendAdToCreative(mockAdObject, remoteDomain, source);
-      expect(JSON.parse(source.postMessage.args[0][0]).ad).to.equal('<script src="http://prebid.org/creative/1.00"></script>');
-      expect(JSON.parse(source.postMessage.args[0][0]).adUrl).to.equal('http://creative.prebid.org/1.00');
+      _sendAdToCreative(mockAdObject, event);
+      expect(JSON.parse(event.source.postMessage.args[0][0]).ad).to.equal('<script src="http://prebid.org/creative/1.00"></script>');
+      expect(JSON.parse(event.source.postMessage.args[0][0]).adUrl).to.equal('http://creative.prebid.org/1.00');
```

Comment:
Should it not be https always now, I know for DFP / GAM served creatives we made a change at one point to always make it secure from the PUC to Prebid.js, is that just a one way thing?

Reply:
it will depend on the event source (`event.source`), so if it comes from a non-secure page, it would be non secure.    We could change the `mockAdObject` in the test to use secure URLs, but I don't think that's pertinent to what's being tested here.

Reply:
Alright sounds good!